sources/light_client.move

```diff
@@ -172,84 +280,81 @@ public fun latest_block(c: &LightClient): &LightBlock {
 /// tx_index is index of transaction in block.
 /// We use little endian encoding for all data.
 public fun verify_tx(
-    c: &LightClient,
+    lc: &LightClient,
     height: u64,
     tx_id: vector<u8>,
     proof: vector<vector<u8>>,
     tx_index: u64
 ): bool {
     // TODO: update this when we have APIs for finalized block.
     // TODO: handle: light block/block_header not exist.
-    let block_hash = c.get_block_hash_by_height(height);
-    let header = c.get_light_block_by_hash(block_hash).header();
+    let block_hash = lc.get_block_hash_by_height(height);
+    let header = lc.get_light_block_by_hash(block_hash).header();
     let merkle_root = header.merkle_root();
     verify_merkle_proof(merkle_root, proof, tx_id, tx_index)
 }
 
-public fun params(c: &LightClient): &Params{
-    &c.params
-}
-
-public fun client_id(c: &LightClient): &UID {
-    &c.id
+public fun params(lc: &LightClient): &Params{
+    &lc.params
 }
 
-public fun client_id_mut(c: &mut LightClient): &mut UID {
-    &mut c.id
+public fun client_id(lc: &LightClient): &UID {
+    &lc.id
 }
 
-public fun relative_ancestor(c: &LightClient, lb: &LightBlock, distance: u64): &LightBlock {
+public fun relative_ancestor(lc: &LightClient, lb: &LightBlock, distance: u64): &LightBlock {
     let ancestor_height = lb.height() - distance;
-    let ancestor_block_hash = c.get_block_hash_by_height(ancestor_height);
-    c.get_light_block_by_hash(ancestor_block_hash)
+    let ancestor_block_hash = lc.get_block_hash_by_height(ancestor_height);
+    lc.get_light_block_by_hash(ancestor_block_hash)
 }
 
-// last_block is a new block that we are adding. The function calculates the required difficulty for the block
-// after the passed the `last_block`.
-public fun calc_next_required_difficulty(c: &LightClient, last_block: &LightBlock, new_block_time: u32) : u32 {
+/// The function calculates the required difficulty for the block after passing `new_block`.
```

Comment:
This is more accurate if we say "for the block after the last_block". Because in insert_header we call this function with the lastest block in the chain/forks.

Reply:
ok, updated